IzvorniKod/frontend/src/components/SearchComponent/SearchComponent.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Input } from "antd";
+import {
+  PaperClipIcon,
+  PaperAirplaneIcon,
+  CpuChipIcon,
+} from "@heroicons/react/24/solid";
+import { cn } from "../../utils/cn.util";
+
+const SearchComponent: React.FC = () => {
+  const [answerContent, setAnswerContent] = useState("");
+  const [isAIEnabled, setIsAIEnabled] = useState(false);
+
+  const handleIconClick = () => {
+    // Handle the logic for adding images here
+    console.log("Icon clicked to add images");
+  };
+
```

Comment:
mislim da nam ova funkcija ne treba ako fileove obradujemo u `handleFileChange` funkciji

Reply:
u toj funkciji ih ucitavamo, ne saljemo ih na be